app/services/security/csrf/__test__/createSessionWithCsrf.test.ts

```diff
@@ -0,0 +1,71 @@
+import { randomBytes } from "node:crypto";
+import { createSession } from "react-router";
```

Comment:
VSCode needs the imports from vitest...
```suggestion
import { createSession } from "react-router";
import { it, vi } from "vitest";
```